tests/utils.ts

```diff
@@ -1,10 +1,9 @@
 import prisma from "./setup";
 
 export const cleanup = async () => {
-  await prisma.message.deleteMany();
+  await prisma.repairRequestImage.deleteMany();
 };
 
 export const teardown = async () => {
-  await cleanup();
-  await prisma.$disconnect();
+  await cleanup().then(() => prisma.$disconnect());

```

Comment:
Nitpick: I'm not fussed but I reckon we could just use `await` instead of mixing the two